src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -166,8 +181,42 @@ public void informSubscriber(String aggregatedObject, JsonNode subscriptionJson)
     }
 
     /**
-     * This method saves the missed Notification into a single document along with
-     * Subscription name, notification meta and time period.
+     * This method reformats notificationMeta and fetches values if applicable
+     * for the requested parameters.
+     *
+     * @param aggregatedObject
+     * @param notificationMeta
+     * @return String
+     */
+    private String reformatNotificationMeta(String aggregatedObject, String notificationMeta) {
+        String URL = notificationMeta.split("\\?")[0];
+
+        List<NameValuePair> params = null;
+        try {
+            params = URLEncodedUtils.parse(new URI(notificationMeta), Charset.forName("UTF-8"));
+        } catch (Exception e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+
+        List<NameValuePair> processedParams = new ArrayList<>();
+        for (NameValuePair param : params) {
+            String name = param.getName(), value = param.getValue();
+            LOGGER.debug("Old: " + name + " : " + value);
```

Comment:
Can you be more clear here. "Old" and "New" don't say that much in the logs.

Reply:
Fixed in comming commit.